upload/tests/views/test_reports.py

```diff
@@ -100,6 +114,7 @@ def test_reports_post_no_auth(db, mocker):
     repository = RepositoryFactory(
         name="the_repo", author__username="codecov", author__service="github"
     )
+    repository.private = False
     token = "BAD"
     commit = CommitFactory(repository=repository)
```

Comment:
the test never calls `commit.save()`. is it failing for the reason we expect, which is an invalid token? or is it failing because the commit isn't in the DB?